caasp-stack.yaml

```diff
@@ -336,7 +336,7 @@ resources:
 
                 chpasswd:
                   list: |
-                    root:linux
+                    root:$root_password
```

Comment:
`$root_password` would need to be added in the params section too.

Reply:
It is already there https://github.com/SUSE/caasp-openstack-heat-templates/blob/master/caasp-stack.yaml#L68

Reply:
It needs to go here: https://github.com/SUSE/caasp-openstack-heat-templates/blob/master/caasp-stack.yaml#L345

Reply:
Ahhh, I'm sorry,  you are right. I fixed it